entries/i/impots.gouv.fr.json

```diff
@@ -2,16 +2,15 @@
   "impots.gouv.fr": {
     "domain": "impots.gouv.fr",
     "url": "https://www.impots.gouv.fr",
-    "contact": {
-      "facebook": "Direction-générale-des-Finances-publiques-250996308359760",
-      "twitter": "dgfip_officiel",
-      "language": "fr"
-    },
+    "tfa": [
+      "sms"
+    ],
+    "documentation": "https://www.impots.gouv.fr/particulier/questions/comment-renforcer-la-securite-de-lacces-mon-espace-particulier",
```

Comment:
What's described sounds more like authorization for specific user actions than authentication at the login phase.
We distinguish between the two in the section about authorization in [Contributing.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#authorization).
Please confirm if this is the case or if 2FA is also prompted on login.

Reply:
I confirm you are right, sorry Carl

Reply:
pas de soucis! 🙂
You've changed the following sites:
* caisse-epargne.fr
* fortuneo.fr
* impots.gouv.fr
* labanquepostale.fr
* lcl.fr
* orange.fr
* sosh.fr

Which of these sites support two-factor authentication and not just two-step authorization?